src/agent/onefuzz-task/src/tasks/merge/generic.rs

```diff
@@ -33,13 +36,28 @@ pub struct Config {
     pub target_options_merge: bool,
     pub tools: SyncedDir,
     pub input_queue: Url,
-    pub inputs: SyncedDir,
+    pub inputs: SyncedDir, // Is this something we can pass to the expander?
```

Comment:
Not sure if this is an old comment but: it's part of `Config`, so yes when you `impl GetExpand for Config` and define a function that takes `&self`, you have access to the properties of the struct referenced by `self`.

Reply:
Sorry, that's a bad comment. What I should have written is: is there / what is the expander variable that that field corresponds to?

Reply:
Might be this one? https://github.com/microsoft/onefuzz/blob/679309a87a27251ffabf698fdd9eae2c643467b6/docs/command-replacements.md?plain=1#L14

Reply:
The only thing that gives me pause is that `supervisor.rs` has a similarly named and typed field in it's config that is used to expand the `input_corpus`, but this config (`merge/generic.rs`) is already providing a value for that (`input_corpus`) expansion variable.
But in the context of being in the merge task, `inputs` corresponding to `generated_inputs` does make sense I think.